CodeEdit/Features/Settings/Models/ModelNameToSettingName.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ModelNameToSettingName.swift
+//  CodeEdit
+//
+//  Created by Raymond Vleeshouwer on 24/06/23.
+//
+
+import Foundation
+import AppKit
+
+struct ModelNameToSettingName {
```

Comment:
I get a crash here when opening, but I think this might be unnecessary. Instead of defining these here, define them on each page struct. That'll allow each struct to supply the localized string from the model instead of a standalone struct. You could define a protocol and make it available on a static property like:

```swift
protocol SearchableSettingsPage {
    static var searchKeys: [String] { get }
}

struct TextEditingSettings: Codable, Hashable, SearchableSettingsPage {
    // ...

    static var searchKeys: [String] {
        [
            "/* Search keys here */"
        ].map { NSLocalizedString($0, comment: "") }
    }
}
```

That'll be much easier to keep track of when updating/adding settings in the future.